optimum/exporters/ipex/model_config.py

```diff
@@ -0,0 +1,98 @@
+#  Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Optional, Tuple
+
+from optimum.exporters.onnx.model_configs import (
+    FalconOnnxConfig,
+    GPT2OnnxConfig,
+    LlamaOnnxConfig,
+)
+from optimum.utils import DEFAULT_DUMMY_SHAPES
+from optimum.utils.input_generators import DummyPastKeyValuesGenerator, DummyTextInputGenerator
+from optimum.utils.normalized_config import NormalizedTextConfig
+
+
+DEFAULT_DUMMY_SHAPES["batch_size"] = 1
```

Comment:
IPEX OP only support the following input:
1. (input_ids = 1; kv-cache >= 0; batch_size >=1)
2. (input_ids >= 1; kv-cache = 0; batch_size >=1)
3. (input_ids > 1; kv-cache > 0; batch_size = 1)

We currently use the 3rd input types.

The 3rd input type is the real case in assisted decoding. If we use the original `DEFAULT_DUMMY_SHAPES`, the dummy input will be the 4th input type (input_ids > 1; kv-cache > 0; batch_size > 1) which will cause core dump of IPEX OP. The 4th input type will not appear in any cases because assisted decoding only supports batch_size=1 in transformers. 

Besides, I have tested the 3rd input type will not have any impact on the performance of IPEXModel greedy_search and beam_search and other decoding methods.